doc-content/enterprise-only/processes/configuring-emails-in-task-notification-proc.adoc

```diff
@@ -0,0 +1,28 @@
+[id='configuring-emails-in-task-notification-proc']
+
+= Configuring emails in task notification
+
+Earlier it was possible to send notifications only to users or group of users in {CENTRAL}. Now you can directly add any email addresses as well.
+
+.Prerequisites
+
+You have created a project in {CENTRAL} and it contains at least one business process asset.
+
+.Procedure
+
+. Create a business process.
+. In the process designer, select the *Activities* tool from the tool palette.
+. Select *User*.
```

Comment:
I believe selecting user is not required, we can directly ask the user to drag and drop the user task.